src/graphql/trusted_domain.rs

```diff
@@ -106,9 +149,25 @@ async fn load(
     super::load_edges(&map, after, before, first, last, EmptyFields)
 }
 
+fn is_valid_domain(domain: &str) -> bool {
+    let domain_regex =
+        Regex::new(r"^(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\.)+[a-zA-Z]{2,}$").unwrap();
+    domain_regex.is_match(domain)
+}
```

Comment:
While I think running a validation of inputs would add value to our program, I have two concerns.
One is that I think domain validation should be applied consistently across trusted domain insertion and update. This code currently only applies domain validation to update. The other is that I think we need a discussion on this validation logic. I personally regard this `is_valid_domain` function as your suggestion. So instead of making a decision on validation logic within this PR, I would like to suggest bringing attention to the discussion by posting a new issue in this repository, and we can leave a todo comment in the code that refers the number of that issue. I believe that would facilitate the process of this PR. Please let me know how you think about this.

Reply:
As mentioned above, I will proceed with the problem below.
https://github.com/aicers/review-web/issues/362